live-examples/html-examples/input/date.html

```diff
@@ -0,0 +1,14 @@
+<fieldset>
+    <legend>Choose trip dates</legend>
+
+    <input type="date"
+           id="start" name="trip" value="2018-07-22"
+           min="2018-01-01" max="2018-12-31">
+    <label for="start">Start</label><br/>
```

Comment:
Nit: For these it seems a little odd to have the label after the input field. Thoughts?

Reply:
Yes, sounds good.